hpctestlib/apps/amber/__init__.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2016-2021 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
+# ReFrame Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import reframe as rfm
+import reframe.utility.sanity as sn
+import reframe.utility.typecheck as typ
+
+__all__ = ["AmberBaseCheck"]
+
+
+class AmberBaseCheck(rfm.RunOnlyRegressionTest):
+    modules = ['Amber']
```

Comment:
Modules go into the derived class. What if other centres call their modules different?